### PR TITLE
core(seo): add image alt to SEO category

### LINKED ISSUE #8393: SEO Score not affected by img without alt attribute
#### Provide the steps to reproduce
1. Run LH on https://www.w3schools.com/code/tryit.asp?filename=G37S02M71XEE . This page contains this code

```
<!DOCTYPE html>
<html>
<body>

<p>
An image that is a link:
<a href="https://www.w3schools.com">
<img src="smiley.gif" width="42" height="42" border="0">
</a>
</p>

</body>
</html>
```

2. Search for SEO score
3. No warning about the missing alt attribute 

#### What is the current behavior?

LH say nothing about img without alt tag

#### What is the expected behavior?

LH should warn about img without alt tag . Google in this document https://support.google.com/webmasters/answer/114016?hl=en says to use descriptive alt text

#### Environment Information
* Affected Channels: DevTools 
* Lighthouse version: 4.0.0
* Node.js version: 6.4.1
* Operating System: updated windows 10

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Seems like a good change to sweep in with breaking changes of a major version :)

**Related Issues/PRs**
fixes #8393

### CODE REVIEW

File: proto/sample_v2_round_trip.json [hunk 1]
```diff
@@ -3614,11 +3619,10 @@
             "warningAuditsGroupTitle": "Passed audits but with warnings", 
             "warningHeader": "Warnings: "
         }
-    },
+    }, 
     "lighthouseVersion": "4.3.0", 
     "requestedUrl": "http://localhost:10200/dobetterweb/dbw_tester.html", 
-    "runWarnings": [],
-    "stackPacks": [],
```
Comment (patrickhulce) [inline proto/sample_v2_round_trip.json]: this happened on update:sample-json, is that bad @exterkamp ?

Comment (patrickhulce) [reply proto/sample_v2_round_trip.json]: based on appveyor it appears the answer is, yes this is bad :)

Comment (exterkamp) [reply proto/sample_v2_round_trip.json]: Yes this is bad :smile: That is uh, slightly broken right now.  Revert the stack packs line.

Comment (brendankenny) [reply proto/sample_v2_round_trip.json]: this is #8245

Comment (patrickhulce) [reply proto/sample_v2_round_trip.json]: oh right I even commented on that 😆 it's been a week...

**Pull request merged**